src/commons/scss/main.scss

```diff
@@ -114,8 +114,8 @@
 
 .disable-pointer {
     pointer-events: none;
-    color: $color-gray;
+    color: $color-gray !important;
```

Comment:
even if we change the color I think a user can still click it.

Isn't it?

Reply:
@cloudbehl the pointer-events has been set to none. So, no user can't click on it.